modules/xatlas_unwrap/register_types.cpp

```diff
@@ -160,9 +160,12 @@ bool xatlas_mesh_lightmap_unwrap_callback(float p_texel_size, const float *p_ver
 
 	const xatlas::Mesh &output = atlas->meshes[0];
 
-	*r_vertices = (int *)malloc(sizeof(int) * output.vertexCount);
-	*r_uvs = (float *)malloc(sizeof(float) * output.vertexCount * 2);
-	*r_indices = (int *)malloc(sizeof(int) * output.indexCount);
+	*r_vertices = (int *)memalloc(sizeof(int) * output.vertexCount);
+	ERR_FAIL_NULL_V_MSG(*r_vertices, false, "Out of memory.");
+	*r_uvs = (float *)memalloc(sizeof(float) * output.vertexCount * 2);
+	ERR_FAIL_NULL_V_MSG(*r_uvs, false, "Out of memory.");
+	*r_indices = (int *)memalloc(sizeof(int) * output.indexCount);
+	ERR_FAIL_NULL_V_MSG(*r_indices, false, "Out of memory.");
```

Comment:
These are members of `xatlas::Mesh` which is in thirdparty code and doesn't use `memfree()`, so I'm not sure they should be changed to `memalloc()`.

Reply:
Reverted to `malloc`, and reverted to normal `free`'s in `scene/resources/mesh.cpp`

Reply:
I think there was confusing here, `xatlas::Mesh` is not related to `scene/resources/mesh.cpp`. `xatlas` is a thirdparty library in `thirdparty/xatlas/`, and it has its own `Mesh` struct.